src/docs/03-Code Guidelines/05-Browser-Support.md

```diff
@@ -1,12 +1,32 @@
 All contents in YouSee DNA is build for and tested in the browsers below. Any YouSee site or web application must be tested in the same browsers.
 
 ## Desktop
-- Internet Explorer 11
-- Chrome (2 latest versions) for Windows and MacOS
-- Firefox (2 latest versions) for Windows and MacOS
-- Safari (2 latest versions) for MacOS
-- Edge (2 latest versions) for Windows
-
-## Mobile/Tablet
-- Chrome (2 latest versions) for iOS and Android
-- Safari (2 latest versions) for iOS
+- Chrome (3 latest versions) for Windows, MacOS and Ubuntu <sup>[[1]](#ref1)</sup>
+- Firefox (3 latest versions) for Windows, MacOS and Ubuntu <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for MacOS <sup>[[2]](#ref2)</sup>
+- Edge (3 latest versions) for Windows <sup>[[1]](#ref1)</sup>
+
+
+## Phone & Tablet
+- Chrome (3 latest versions) for Android and iOS <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for iOS <sup>[[2]](#ref2)</sup>
+- Samsung Internet (latest version) for Android <sup>[[3]](#ref3)</sup>
+
+
+## Partial support
+- Internet Explorer 11 for Windows <sup>[[4]](#ref4)</sup>

```

Comment:
I don't think this is acceptable from a business standpoint. YouSee still have a lot of active IE11 users (8%, I think) and I think that requires full support.

Reply:
We have agreed to stop full support for Internet Explorer 11. It still needs to work so we should not start to get incidents from customers. But if there is a few pixels off there and there we won't fix it. It has to look acceptable of cause but not perfect.

Reply:
I think you should move `Internet Explorer 11` up to desktop browsers we support and then write (only partial support) and then at line 16 write what partial support is and even come up with an example maybe.